### PR TITLE
docs: fix README sections order

### PR DESCRIPTION
**Pull request by agnes512**
Fix the order to make it consistent with other connector README files

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-mongodb) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -494,3 +433,64 @@ export class User extends Entity {
 ## Release notes
```
Comment (dhmlau) [inline README.md:433]: maybe move the release notes at the bottom.

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
The change is unrelated to bluemix-security@develop failure.

**Pull request merged**